exchanges/bybit/bybit_wrapper.go

```diff
@@ -275,7 +295,11 @@ func (by *Bybit) Run(ctx context.Context) {
 		return
 	}
 
-	err := by.UpdateTradablePairs(ctx, false)
+	err := by.RetrieveAndSetAccountType(ctx)
```

Comment:
ValidateAPICredentials() needs this (by.AccountType), as it is used in UpdateAccountInfo(). However, ValidateAPICredentials() is called before we get here (see engine.go@844:861). Maybe move this to inside ValidateAPICredentials(), or to somewhere else before that?
(This prevents unified accounts from using their API key)

Reply:
bump @samuael